cpuid/src/bit_helper.rs

```diff
@@ -32,6 +32,8 @@ pub struct BitRange {
     pub lsb_index: u32,
 }
 
+/// Trait containing helper methods for BitRange
```

Comment:
nit:
```suggestion
/// Trait containing helper methods for [`BitRange`](struct.BitRange.html).
```
This should generate a hyperlink in the documentation.

Reply:
Done